getting-started/main.cpp

```diff
@@ -143,32 +133,29 @@ static void import_a_key(const uint8_t *key, size_t key_len)
 
     /* Destroy the key */
     psa_destroy_key(handle);
-
-    mbedtls_psa_crypto_free();
 }
 
 static void sign_a_message_using_rsa(const uint8_t *key, size_t key_len)
 {
     psa_status_t status;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
-    uint8_t hash[] = "INPUT_FOR_SIGN";
-    uint8_t signature[PSA_ASYMMETRIC_SIGNATURE_MAX_SIZE] = {0};
+    psa_algorithm_t alg = PSA_ALG_RSA_PKCS1V15_SIGN(PSA_ALG_SHA_256);
+    // SHA-256 of {'a', 'b', 'c'}
+    const uint8_t hash[] = {
```

Comment:
I'd recommend keeping the size parameter here, to make the code a bit more robust (if a few elements got removed on accident, for example, we'd not read uninitialzed memory). I'd also recommend "static" to ensure we don't use stack memory too much.

`static const uint8_t hash[PSA_HASH_SIZE(PSA_ALG_SHA_256)]`

Reply:
This would be a general improvement that also applies to other examples in this application. I can add an extra commit for this.

Reply:
(Most others already have `static`, "verify_a_hash" where I copied the hash just happened to not have it 😮 )